### ISSUE TITLE
Add Free2Move's trip data to SMRT Tracker

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#8091]
> What application are you using?

Shared Mobility Operations

> Describe the problem.

I need Free2Move's trip data backfilled for Dec in the SMRT Tracker.

> Describe the outcome you'd like to see when this feature is implemented.

I should be able to see the trip data for Free2Move in the SMRT Tracker

> Describe any workarounds you currently have in place or alternative solutions you've considered.

There is no workaround for pulling this data ingested by COA.

> Is there anything else we should know?

Jan 7, 2022.

> Requested By
Andrea M.

Request ID: DTS22-103265

**Comment by patrickm02L** [comment]
Sergio says we need credentials and API endpoint where we can extract the data.

**Comment by patrickm02L** [comment]
@sergiogcx thanks for working with Free2Move via email. Can you please update progress you're making here?

**Comment by sergiogcx** [comment]
Updates:

Good: I am able to retrieve the JWT token. I tried the credentials they provided, and the way their oauth endpoint works is different (OAuth + Basic Auth). So I managed to retrieve the token and modified the mds code to allow for their unique process (not a lot of changes, only a few lines).

Bad: The token does not work, their endponts reject their own token. I've been working on this for 4 hours already (it should have taken 15 minutes).

Unfortunately, I cannot proceed until they provide some clarity. I responded on the email thread with the errors I am seeing, hopefully they will reach back soon.
